src/Azure.Functions.Cli/Actions/LocalActions/InitAction.cs

```diff
@@ -424,33 +423,20 @@ private static async Task WriteHostJson(WorkerRuntime workerRuntime, bool manage
 
             if (workerRuntime == Helpers.WorkerRuntime.powershell && managedDependenciesOption)
             {
-                hostJsonContent = await AddManagedDependencyConfig(hostJsonContent);
+                hostJsonContent = await hostJsonContent.AppendContent(StaticResources.ManagedDependenciesConfig, Constants.ManagedDependencyConfigPropertyName);
             }
 
             if (extensionBundle)
             {
-                hostJsonContent = await AddBundleConfig(hostJsonContent);
+                hostJsonContent = await hostJsonContent.AppendContent(StaticResources.BundleConfig, Constants.ExtensionBundleConfigPropertyName);
             }
 
-            await WriteFiles("host.json", hostJsonContent);
-        }
-
-        private static async Task<string> AddBundleConfig(string hostJsonContent)
-        {
-            var hostJsonObj = JsonConvert.DeserializeObject<JObject>(hostJsonContent);
-            var bundleConfigContent = await StaticResources.BundleConfig;
-            var bundleConfig = JsonConvert.DeserializeObject<JToken>(bundleConfigContent);
-            hostJsonObj.Add(Constants.ExtensionBundleConfigPropertyName, bundleConfig);
-            return JsonConvert.SerializeObject(hostJsonObj, Formatting.Indented);
-        }
+            if(workerRuntime == Helpers.WorkerRuntime.custom)
+            {
+                hostJsonContent = await hostJsonContent.AppendContent(StaticResources.CustomHandlerConfig, Constants.CustomHandlerPropertyName);
```

Comment:
nit: `Helpers.WorkerRuntime.Custom` ? casing on custom

Reply:
The rest of the values in the enum are fully lowercased so just stuck with that.